x-pack/test_serverless/functional/test_suites/observability/config.screenshots.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createTestConfig } from '../../config.base';
+
+const enabledActionTypes = ['.index', '.server-log'];
+
+export default createTestConfig({
+  serverlessProject: 'oblt',
+  testFiles: [require.resolve('./screenshot_creation')],
+  kbnServerArgs: [`--xpack.actions.enabledActionTypes=${JSON.stringify(enabledActionTypes)}`],
+  junit: {
+    reportName: 'Serverless Observability Screenshot Creation',
+  },
+
+  // include settings from project controller
+  // https://github.com/elastic/project-controller/blob/main/internal/project/esproject/config/elasticsearch.yml
```

Comment:
I see this was copied over from another file so I'm not going to block on it, but we should follow up with removing these links.  Generally we don't want to link from public repositories to private repositories.

Reply:
Done in c6fe3d021d8be6873610f85505d0acf1f29af66f